app/bundles/CampaignBundle/Tests/Controller/CampaignControllerTest.php

```diff
@@ -13,7 +13,7 @@ class CampaignControllerTest extends MauticMysqlTestCase
     public function testIndexActionWhenNotFiltered(): void
     {
         $this->client->request('GET', '/s/campaigns');
-        $clientResponse = $this->client->getResponse();
+        $this->client->getResponse();
```

Comment:
This file has all sorts of unused method calls.

Reply:
As for me this has nothing to do with the symfony6 update, so this either need to be removed from the PR, or be fixed.

Reply:
I see what you mean. I'll remove the unnecessary method calls.